app/src/main/java/org/simple/clinic/reports/ReportsRepository.kt

```diff
@@ -49,5 +49,11 @@ class ReportsRepository @Inject constructor(
           .ignoreElement()
 
   fun deleteReportsFile(): Single<DeleteFileResult> =
```

Comment:
This has been fixed because the method has been rewritten. However, this is not an extra line and is intentional. We are following this convention where an `if` block with a corresponding `else` block has an extra line at the end for better separation between the two blocks.